hello_world.py

```diff
@@ -1 +1 @@
-echo ("Hello, World!")
+print "Hello, World!"
```

Comment:
Thanks for this, @jasonrudolph. While you're in here, would you mind making this message even more personal? Something like: "Hello, Python Namibia!" :snake: :earth_africa: